### DISCUSSION TITLE
Trouble adding Spezi package

### DISCUSSION THREAD

**Discussion by jungenwu** [StanfordSpezi/.github#53] [Challenges and Help] [answered]
### What Stanford Spezi module is your challenge related to?

Spezi

### Description

<img width="1094" alt="Screen Shot 2024-06-20 at 11 35 16 PM" src="https://github.com/StanfordSpezi/.github/assets/9994781/e1935b64-d2bc-4c0f-9b1e-cc0ee47e86e7">

Got this message - file 'package.resolved' is corrupted. I am new to coding so please bear with me on the fundamental questions :) 

### Reproduction

1. Xcode -> add package dependencies 
2. Copy the Spezi Github URL 
3. Got the message above 
4. Tried to run 'LLMonFHIR' and got the message saying that the package was not installed properly. 

### Expected behavior

Speiz package should be added to the LLMonFHIR with no issue. 

### Code of Conduct

- [X] I agree to follow this projects's Code of Conduct and Contributing Guidelines

**Comment by vishnuravi** [comment]
Hi @jungenwu, thanks for trying LLMonFHIR! You don't need to add the Spezi package to LLMonFHIR, as it is already included in the project.

**Comment by PSchmiedmayer** [comment, marked as answer]
@jungenwu Did you resolve the issue? Sometimes it might also be an issue with:
1. Updating Xcode to the latest version
2. Completely removing the Package.resolved file within the Xcode Project container and re-open the project after that to re-resolved the dependencies
3. Reset the package dependency cache using File > Packages > Reset Package Caches

**Reply by jungenwu** [reply]
Thanks a lot, Paul. 

Yes, it was indeed the Xcode version issue. It took me a while to identify this issue. 

I got that resolved a few days ago, and sorry about following back on this thread. 

Our team is excited about using this framework. Exciting stuff! Plan to contribute to this project too. 

Cheers!

**Reply by PSchmiedmayer** [reply]
Sounds amazing; happy to hear that this helped @jungenwu! Good luck with your project and we would love to hear what you are working on!

We aim to create a community of people sharing their projects in GitHub discussions so feel free to share your work there: https://github.com/orgs/StanfordSpezi/discussions/categories/show-and-tell 🚀
